src/Dap.Protocol/Requests/SetBreakpointsArguments.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using OmniSharp.Extensions.DebugAdapter.Protocol.Models;
+using OmniSharp.Extensions.DebugAdapter.Protocol.Serialization;
+using OmniSharp.Extensions.Embedded.MediatR;
+
+namespace OmniSharp.Extensions.DebugAdapter.Protocol.Requests
+{
+    public class SetBreakpointsArguments : IRequest<SetBreakpointsResponse>
+    {
+        /// <summary>
+        /// The source location of the breakpoints; either 'source.path' or 'source.reference' must be specified.
+        /// </summary>
+        public Source Source { get; set; }
+
+        /// <summary>
+        /// The code locations of the breakpoints.
+        /// </summary>
+        [Optional] public Container<SourceBreakpoint> Breakpoints { get; set; }
+
+        /// <summary>
+        /// Deprecated: The code locations of the breakpoints.
+        /// </summary>
+        [Obsolete("Deprecated")]
+        [Optional] public Container<long> Lines { get; set; }
```

Comment:
@david-driscoll @TylerLeonhardt This seems to have been written deprecated...what should be used in place of it?